policy/modules/system/systemd.if

```diff
@@ -1136,6 +1136,25 @@ interface(`systemd_dbus_chat_homed',`
 	allow systemd_homed_t $1:dbus send_msg;
 ')
 
+########################################
+## <summary>
+##   Connect to /run/systemd/userdb/io.systemd.Home to
+##   query user account information.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`systemd_stream_connect_homed',`
+	gen_require(`
+		type systemd_homed_t;
+	')
+
+	allow $1 systemd_homed_t:unix_stream_socket connectto;
```

Comment:
The interface summary says this is a named socket, so there should be `dir` and `sock_file` access too, preferably using `stream_connect_pattern()`.

Reply:
Will do

Reply:
Done